std/stdio.d

```diff
@@ -725,7 +725,8 @@ $(D rawRead) always reads in binary mode on Windows.
     {
         import std.exception : enforce, errnoEnforce;
 
-        enforce(buffer.length, "rawRead must take a non-empty buffer");
+        if (!buffer.length)
+            enforce(false, "rawRead must take a non-empty buffer");
```

Comment:
Thanks, will do. Any idea why enforce does not get inlined? The body just consists of `if (!value) throw new Exception(msg)`